trunk/SUAVE/Methods/Propulsion/propeller_design.py

```diff
@@ -165,41 +145,25 @@ def propeller_design(prop,number_of_stations=20):
         Ma      = Wc/speed_of_sound
         RE      = Wc/nu
 
-        #Step 4, determine epsilon and alpha from airfoil data  
-        alpha = np.zeros_like(RE)
-        Cdval = np.zeros_like(RE)  
-        for i in range(N):
-            AoA_old_guess = 0.01
-            cl_diff       = 1  
-            broke         = False   
-            ii            = 0
+        # Step 4, determine epsilon and alpha from airfoil data  
+        if airfoil_flag:   
+            # compute airfoil polars for airfoils 
+            airfoil_polars  = compute_airfoil_polars(a_geo, a_pol)  
+            airfoil_cl_surs = airfoil_polars.lift_coefficient_surrogates 
+            airfoil_cd_surs = airfoil_polars.drag_coefficient_surrogates 
+            
+            alpha = np.zeros_like(RE)
+            Cdval = np.zeros_like(RE)  
+            
+            for i in range(N):  
```

Comment:
Does this need to be in a for loop? A newton solver should be able to do this is as a vector.